package.json

```diff
@@ -22,16 +22,17 @@
     "test": "grunt test"
   },
   "dependencies": {
+    "amqplib": "^0.5.1",
     "async": "1.5.2",
+    "body-parser": "1.15.0",
     "dateformat": "1.0.12",
     "express": "^4.11.2",
-    "body-parser": "1.15.0",
-    "amqplib": "^0.5.1",
-    "iotagent-node-lib": "2.6.x",
+    "iotagent-node-lib": "https://github.com/dcalvoalonso/iotagent-node-lib.git#task/activeAttsNgsi2",
     "logops": "1.0.0-alpha.7",
-    "mqtt": "1.7.0",
-    "request": "^2.69.0",
+    "moment": "^2.20.1",
```

Comment:
If `moment` is used only in unit test, then it should be moved to the `devDependencies` section, isn't it?

Reply:
Fixed in 9f824541d71e57bf5a4532db6953e6e81cf96605